docs/development/getting-started.rst

```diff
@@ -23,7 +23,7 @@ automatically, so all you have to do is:
 
 .. code-block:: console
 
-    $ py.test
+    $ tox
```

Comment:
I think `tox -e py` makes more sense, or at least `tox --skip-missing-interpreters`, since it's unlikely all Python versions will be available.

Reply:
updated the PR